scipy/stats/_discrete_distns.py

```diff
@@ -514,6 +514,121 @@ def _logcdf(self, k, M, n, N):
 hypergeom = hypergeom_gen(name='hypergeom')
 
 
+class nhypergeom_gen(rv_discrete):
+    r"""A negative hypergeometric discrete random variable.
+
+    The negative hypergeometric distribution models the probability
+    of :math:`k` successes in a sample of exactly :math:`r` failures
+    taken without replacement where :math:`N` are the total number
+    of elements and :math:`K` are defined to be successes.
+
+    It is equivalent to observing :math:`k` successes in :math:`k+r-1`
+    samples with :math:`k+r`'th sample being a failure. The former
+    can be modelled as a hypergeometric distribution. Refer
+    `hypergeom` distribution for more information.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The symbols used to denote the shape parameters (`N`, `K`, and `r`) are not
+    universally accepted. See the Examples for a clarification of the
+    definitions used here.
+
+    The probability mass function is defined as,
+
+    .. math:: f(k; N, K, r) = \frac{{{k+r-1}\choose{k}}{{N-r-k}\choose{K-k}}}
+                                   {{N \choose K}}
+
+    for :math:`k \in [0, K]` and the binomian co-efficient is:
+
+    .. math:: \binom{n}{k} \equiv \frac{n!}{k! (n - k)!}.
+
+    %(after_notes)s
+
+    Examples
+    --------
+    >>> from scipy.stats import nhypergeom
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+
+    Suppose we have a collection of 20 animals, of which 7 are dogs.
+    Then if we want to know the probability of finding a given number
+    of dogs (successes) in a sample with exactly 12 animals that
+    aren't dogs (failures), we can initialize a frozen distribution
+    and plot the probability mass function:
+
+    >>> N, K, r = [20, 7, 12]
+    >>> rv = nhypergeom(N, K, r)
+    >>> x = np.arange(0, K+2)
+    >>> pmf_dogs = rv.pmf(x)
+
+    >>> fig = plt.figure()
+    >>> ax = fig.add_subplot(111)
+    >>> ax.plot(x, pmf_dogs, 'bo')
+    >>> ax.vlines(x, 0, pmf_dogs, lw=2)
+    >>> ax.set_xlabel('# of dogs in our group with given 12 failures')
+    >>> ax.set_ylabel('nhypergeom PMF')
+    >>> plt.show()
+
+    Instead of using a frozen distribution we can also use `nhypergeom`
+    methods directly.  To for example obtain the probability mass
+    function, use:
+
+    >>> prb = nhypergeom.pmf(x, N, K, r)
+
+    And to generate random numbers:
+
+    >>> R = nhypergeom.rvs(N, K, r, size=10)
+
+    See Also
+    --------
+    hypergeom, binom, nbinom
```

Comment:
The distributions `binom`, `nbinom`, `hypergeom`, and `nhypergeom` form a closely linked set. It would be great if you would add See Also sections to those other distributions as part of this PR. 

So in `binom`:
```
See Also
--------
nbinom, hypergeom, nhypergeom
``` 

In `nbinom`:
```
See Also
--------
binom, hypergeom, nhypergeom
```
and in `hypergeom`:
```
See Also
--------
binom, nbinom, nhypergeom
```
Thanks!

Reply:
Done!